code/style.css

```diff
@@ -6,39 +6,58 @@ body {
   margin: 0;
   padding: 0;
   font-family: 'Montserrat', sans-serif;
```

Comment:
I had to. manualy load the css file because it was so big and did not show by itself, this feels so promising!!!